Translations/ja.lproj/Localizable.strings

```diff
@@ -23,9 +23,9 @@
 "Password needed" = "パスワードは必要";
 "File \"%@\" needs a password to be extracted" = "\"%@\" を解凍できるようにパスワードを使用する必要があります。";
 "File \"%@\" needs a special encoding to be extracted" = "File \"%@\" needs a special encoding to be extracted";
-"Extraction of \"%@\" done" = "\"%@\" は解凍されました。";
-"Extraction of \"%@\" failed." = "\"%@\" は解凍できませんでした。";
-"Extraction of %i files done" = "Extraction of %i files done";
+"Extraction of \"%@\" done" = " \"%@\" は解凍されました。";
+"Extraction of \"%@\" failed." = " \"%@\" は解凍できませんでした。";
+"Extraction of %i files done" = " %i の展開が完了しました";
```

Comment:
Why the extra spaces at the beginning?

Reply:
I've removed them, please let me know if they are needed and why. Thanks again @Sh1hu!

https://github.com/aonez/Keka/commit/fafb58794b8e1c3b2a7e6a235bedfda0d2fc31e6

Reply:
That space is a mistake.  sorry.